skeyan/stock-portfolio
# Finish login registration

**Changes:**
- Implemented basic conditional rendering for login and logout
- Updated frontend to keep track of logged-in status (might need more auth)
- Fixed alert messages
- Fixes #9 and fixes #10 
- Minor CSS changes on Portfolio page

**To Test:**
- Run `npm start` and `nodemon index.js` on the front-end and back-end respectively. 
- You should be able to first register and then log in with responsive message alerts. 
- Logging in should redirect you to the homepage immediately, as does logging out.
